unifier/apps/core/management/commands/neoxscan.py

```diff
@@ -0,0 +1,137 @@
+import logging
+
+import requests
+from bs4 import BeautifulSoup
+from bs4.element import ResultSet, Tag
+from django.core.management.base import BaseCommand
+from requests.models import Response
+from rest_framework import status
+from unifier.apps.core.models import Manga, MangaChapter, Platform
+from unifier.apps.core.models.chapter import Language
+from unifier.apps.core.services import BulkCreateMangaChapterService
+
+
+class Command(BaseCommand):
+    help = ""
+    headers = {
+        "User-Agent": (
+            "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 "
+            "(KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36"
+        )
+    }
+
+    def handle(self, *args, **kwargs):
+        self.platform = Platform.objects.get(name="neoxscan")
+        mangas = self.platform.mangas.all()
+
+        if self.platform:
+            for manga in mangas:
+                self.stdout.write(f"Current manga: {manga}")
+
+                chapters_content = []
+
+                manga_url = self._get_manga_url(manga)
+                manga_id = int(self._get_manga_id(manga_url))
+                chapters = self._get_manga_chapters(manga_id)
+                chapters.reverse()
+
+                if self._has_new_chapter(int(chapters[-1].find("a").text.strip().replace("Cap. ", "")), manga):
+                    for chapter in chapters:
+                        if not self._has_new_chapter(int(chapter.find("a").text.strip().replace("Cap. ", "")), manga):
+                            self.stdout.write(f"{manga} don't have any new chapters")
+                            break
```

Comment:
```suggestion
                            continue
```

If u don't find a chapter, you'll break the whole loop? And about the other mangas? Also, the structure is kinda weird. You check if has new chapter, then for every chapter you check if don't has new chapter (?)